cli/src/main/java/com/devonfw/tools/ide/tool/eclipse/Eclipse.java

```diff
@@ -68,14 +67,14 @@ protected ProcessResult runEclipse(ProcessMode processMode, String... args) {
     pc.executable(toolPath);
     Path configurationPath = getPluginsInstallationPath().resolve("configuration");
     this.context.getFileAccess().mkdirs(configurationPath);
+    pc.addArg("-application " + toolPath);
```

Comment:
mhm, did you ever test this (manually)?
The application argument takes a classname like `org.eclipse.equinox.p2.director` as parameter.
Passing `toolPath` here does not seem to make any sense to me and IMHO cannot work.
Also this argument must not be always present - if you launch the regular Eclipse GUI this should be omitted.
See the old  code to understand what needs to happen:
https://github.com/devonfw/ide/blob/cdf4af5de21489f3fdcbfdab41af858822e0d598/scripts/src/main/resources/scripts/command/eclipse#L187

Reply:
I've tested only the start of eclipse and didn't realize that the application param had no effect because the plugins were already installed.
I've adjusted the installPlugin method now and tested the install process after deleting the whole plugins folder in my test project.